hydra-plutus/exe/inspect-script/Main.hs

```diff
@@ -102,7 +103,7 @@ main = do
     , (abortDatum, "abortDatum")
     ]
 
-  headDatum = toData $ Head.Initial 1_000_000_000_000 []
+  headDatum = toData $ Head.Initial 1_000_000_000_000 [] (CurrencySymbol hydraHeadV1)
```

Comment:
This is not what the policy ID will be. It's rather a `hash(mu_head(someTxIn))`.

I think we should not bother and just drop the datum serialization here. It's just examples.